components/expenses/PayoutBankInformationForm.js

```diff
@@ -72,8 +72,109 @@ const requiredFieldsQuery = gqlV2`
   }
 `;
 
-const RequiredFields = ({ disabled, getFieldName, formik, host, currency }) => {
-  const { loading, error, data } = useQuery(requiredFieldsQuery, {
+const Input = props => {
+  const { input, getFieldName, disabled, currency, loading, refetch, formik, host } = props;
+  const fieldName =
+    input.key === 'accountHolderName' ? getFieldName(`data.${input.key}`) : getFieldName(`data.details.${input.key}`);
+
+  if (input.type === 'text') {
+    const validate = input.validationRegexp
+      ? value => (new RegExp(input.validationRegexp).test(value) ? undefined : `Invalid ${input.name}`)
+      : undefined;
+    return (
+      <Box key={input.key} mt={2} flex="1">
+        <FastField name={fieldName} validate={validate}>
+          {({ field, meta }) => (
+            <StyledInputField label={input.name} required error={meta.touched && meta.error}>
+              {() => <Field as={StyledInput} placeholder={input.example} {...field} disabled={disabled} width="100%" />}
```

Comment:
Would be great to pass the `error `prop to the `StyledInput` too so that it displays the red border